### PR TITLE
fixed ulab spectrogram imports

### LINKED ISSUE #2335: ulab.scipy.signal import spectrogram stopped working with 8.0.0-beta.4
### CircuitPython version

```python
Adafruit CircuitPython 8.0.0-beta.4 on 2022-10-30; Adafruit QT Py RP2040 with rp2040
```

### Code/REPL

```python
from ulab.scipy.signal import spectrogram
```

### Behavior

ImportError: can't import name spectrogram

### Description

Had working example of this tutorial: https://learn.adafruit.com/mini-led-matrix-audio-visualizer
Stopped working after latest update.

### Additional information

From the repl >>> help(ulab.scipy.signal) 
shows only sosfilt function. 
None for [scipy.signal.spectrogram](https://micropython-ulab.readthedocs.io/en/2.1.2/scipy-signal.html#spectrogram)

### PR DESCRIPTION
**Pull request by evaherrada**
Fixes #2335

### CODE REVIEW

File: Feather_Sense_Audio_Visualizer_13x9_RGB_LED_Matrix/waterfall_visualizer/code.py [hunk 1]
```diff
@@ -30,34 +34,79 @@
...
+    0xFFC100,
+    0xFFAB00,
+    0xFF9500,
+    0xFF7C00,
+    0xFF6100,
+    0xFF4100,
+    0xFF0000,
+    0xFF0000,
+    0xFF0000,
+]
```
Comment (kattni) [inline Feather_Sense_Audio_Visualizer_13x9_RGB_LED_Matrix/waterfall_visualizer/code.py:90]: Please revert this change. Then disable Black for this section, and reenable it after the end of the section.

File: Ukulele/code.py [hunk 2]
```diff
@@ -71,56 +75,520 @@
...
+        56,
+        48,
+        55,
+        49,
+        54,
+        50,
+        53,
+        51,
+        52,
+    ],
```
Comment (kattni) [inline Ukulele/code.py:191]: Please revert this change. Then disable Black for this section, and reenable it after the end of the section.

File: Ukulele/code.py [hunk 3]
```diff
@@ -71,56 +75,520 @@
...
+        95,
+        96,
+        97,
+        98,
+        99,
+        100,
+        101,
+        102,
+        103,
+    ],
```
Comment (kattni) [inline Ukulele/code.py:303]: Please revert this change. Then disable Black for this section, and reenable it after the end of the section.

File: Ukulele/code.py [hunk 4]
```diff
@@ -71,56 +75,520 @@
...
+        4,
+        100,
+        3,
+        101,
+        2,
+        102,
+        1,
+        103,
+        0,
+    ],
```
Comment (kattni) [inline Ukulele/code.py:416]: Please revert this change. Then disable Black for this section, and reenable it after the end of the section.

File: ulab_Crunch_Numbers_Fast/waterfall/code.py [hunk 5]
```diff
@@ -12,32 +12,81 @@
...
+        0x0064FF,
+        0x0054FF,
+        0x0044FF,
+        0x0032FF,
+        0x0022FF,
+        0x0012FF,
+        0x0002FF,
+        0x0000FF,
+    )
+):
```
Comment (kattni) [inline ulab_Crunch_Numbers_Fast/waterfall/code.py:80]: Please revert this change. Then disable Black for this section, and reenable it after the end of the section.

**Review by kattni — CHANGES_REQUESTED** [review]
@jepler Thank you for the tag. @evaherrada Thanks for the fix! Please follow the instructions below. You should not globally disable Black, so you'll do it for each individual section.  Please apply these instructions to any obvious section I may have missed (I believe I caught them all).

**Comment by kattni** [comment]
@evaherrada CI is failing, but I think I know why.

**Comment by kattni** [comment]
@caternuson Is this the same failure that was happening previously and fixed by a PR? Does Eva simply need to pull in the changes, and let CI rerun?

**Comment by caternuson** [comment]
Yep. Need to pull in the changes added with this PR:
https://github.com/adafruit/Adafruit_Learning_System_Guides/pull/2341

**Comment by kattni** [comment]
@evaherrada Please check out the link Carter provided [here](https://github.com/adafruit/Adafruit_Learning_System_Guides/pull/2336#issuecomment-1332835058). You need to pull in some recent changes to make the CI pass.

**Comment by flavio-fernandes** [comment]
/lgtm 
still an issue on 
```
dafruit CircuitPython 8.2.9 on 2023-12-06; Adafruit LED Glasses Driver nRF52840 with nRF52840
Board ID:adafruit_led_glasses_nrf52840
```

**Pull request merged**